handler.py

```diff
@@ -177,8 +155,8 @@ def putInQueue(event, context):
         "www.mozilla.org",
         "infosec.mozilla.org",
     ]
-    randomizer = Randomizer(target_list)
-    target = Target(name=randomizer.next())
+    hostlist = Hosts(target_list)
+    target = Target(name=hostlist.next())
```

Comment:
If this is fed with target_list, which would have 2 elements ("A" and "B"), does "B" ever get loaded.  I feel like only "A" would ever get put in the queue.  But when I look at the next() implementation, looks like it's random sourcing, so it will eventually work out, but we'll want to change that before prod.

Reply:
This is the manual function - there is no event to trigger it, and it was implemented for test purposes only.

Happy to completely remove it.

Reply:
ah, ok, understood